.expeditor/config.yml

```diff
@@ -0,0 +1,39 @@
+# Documentation available at http://expeditor-docs.es.chef.io/
+
+# Slack channel in Chef Software slack to send notifications about build failures, etc
+slack:
+  notify_channel: sustaining-notify
+
+rubygems:
+  - win32-taskscheduler
+
+github:
+  # This deletes the GitHub PR branch after successfully merged into the release branch
+  delete_branch_on_merge: true
+  # The tag format to use (e.g. v1.0.0)
+  version_tag_format: "win32-taskscheduler-v{{version}}"
+  # The Github Team primarily responsible for handling incoming Pull Requests.
+  maintainer_group: chef/msys-developers
+  release_branch:
+    - ole:
```

Comment:
http://expeditor.es.chef.io/workflows/multiple-branches/

I think this has to be:
```
release_branch:
  - master:
      version_constraint: "0.*"
  - ole:
      version_constraint: "1.*"
```

Then we need to change `VERSION` to 1.0.0.

Reply:
@btm does that mean our next release version fro win32-taskscheduler will be 1.0.0 and not the 0.4.2 since we are releasing from ole branch ?

Reply:
yes, I think we'll end up doing that. 

> By default, Expeditor assumes you have one release branch: master, and that this release branch is used for all your releases. With a simple modification to your .expeditor/config.yml, Expeditor can support the maintenance of multiple release branches. To do this, you use the github.release_branch setting to specify two or more release branches and the version constraints that differentiate those branches.

> One of these release branches must be master.

I read that as we need version constraints, and unless we can make a version constraint like this, we have to flip the major:
```
version_constraint: "< 0.4.0"
version_constraint: ">= 0.4.0"
```

My sense is that we can't do that. You could peruse https://github.com/chef/es-lita but a quick `git grep version_constraint` on the repo isn't promising.

So yes, we will change `VERSION` to 1.0.0 and add the `Expeditor: Skip All` label to this branch.